### PR TITLE
+foo +Bar

### PR DESCRIPTION
**Pull request by k-morozov-kpi**

### CODE REVIEW

File: bar.cpp [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#include "bar.hpp"
+
```
Comment (k-morozov-oop) [inline bar.cpp:2]: Remove empty line

File: bar.hpp [hunk 2]
```diff
@@ -0,0 +1,12 @@
+namespace test
+{
+
+class Bar
+{
+public:
+    Bar(int x);
+    void print();
+    int m_x;
```
Comment (k-morozov-oop) [inline bar.hpp:13]: Make this field private (code style 4.6) [Code style: Access control](https://google.github.io/styleguide/cppguide.html#Access_Control)

File: bar.cpp [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+#include <iostream>
+
+namespace test
+{
+    
+Bar::Bar(int x)
+{
+    m_x = x;
```
Comment (k-morozov-oop) [inline bar.cpp]: Prefer initialization over assignment in constructors.

File: bar.cpp [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+void Bar::print()
```
Comment (k-morozov-oop) [inline bar.cpp]: Make it const

File: bar.hpp [hunk 5]
```diff
@@ -0,0 +1,12 @@
+namespace test
```
Comment (k-morozov-oop) [inline bar.hpp:4]: Use conditional compilation to avoid duplication of include.

File: main.cpp [hunk 6]
```diff
@@ -1,6 +1,17 @@
 #include <iostream>
 
+void foo()
+{
+    std::cout << "Foo 1" << std::endl;
```
Comment (k-morozov-oop) [inline main.cpp]: Use loop to avoid duplication

**Review by k-morozov-oop — CHANGES_REQUESTED** [review]

File: bar.cpp (hunk 1 above)
Comment (k-morozov-kpi) [reply bar.cpp:2]: It is incorrect according to [code style](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)

File: bar.hpp (hunk 2 above)
Comment (k-morozov-kpi) [reply bar.hpp:13]: fixed

File: bar.cpp (hunk 3 above)
Comment (k-morozov-kpi) [reply bar.cpp]: fixed

File: bar.cpp (hunk 4 above)
Comment (k-morozov-kpi) [reply bar.cpp]: fixed

File: bar.hpp (hunk 5 above)
Comment (k-morozov-kpi) [reply bar.hpp:4]: fixed

File: main.cpp (hunk 6 above)
Comment (k-morozov-kpi) [reply main.cpp]: Not needed because exactly 5 iterations is required.

Comment (k-morozov-oop) [reply main.cpp]: Anyway use a loop

**Review by k-morozov-oop — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 6 above)
Comment (k-morozov-kpi) [reply main.cpp]: fixed

**Review by k-morozov-oop — APPROVED** [review]

**Pull request merged**